src/components/claim/RewardOverview.svelte

```diff
@@ -19,9 +19,8 @@
 </script>
 
 <Card
-  title={`Round ${roundInfo?.id}  -  ${
-    parseInt(roundInfo?.passive) + parseInt(roundInfo?.active)
-  } OCEAN`}
+  title={`Round ${roundInfo?.id}`}
```

Comment:
Given that this is the main heading for the whole site I think it makes sense to have "Data Farming" in the title

```suggestion
  title={`Data Farming Round ${roundInfo?.id}`}
```